types/apis/space.cattle.io/v1beta1/publicdomain.go

```diff
@@ -0,0 +1,19 @@
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PublicDomain struct {
```

Comment:
can we make PublicDomain namespaced and store them all in RioSystemNamespace?  I'm trying to not ever make cluster level objects anymore